airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -600,11 +649,21 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session) -
                             )
                             continue
 
-                if executor_obj := self._try_to_load_executor(task_instance.executor):
+                if executor_obj := self._try_to_load_executor(task_instance, session):
                     if TYPE_CHECKING:
                         # All executors should have a name if they are initted from the executor_loader.
                         # But we need to check for None to make mypy happy.
                         assert executor_obj.name
+                    # TODO: move to debug or removed entirely
+                    self.log.info("Using executor %s for task %s", executor_obj.name, task_instance)
+                    task_team = self._get_task_team_name(task_instance, session)
+                    # TODO: move to debug or removed entirely
+                    self.log.info("Task %s belongs to team %s", task_instance, task_team)
+                    if task_team is not None:
+                        executor_team = getattr(executor_obj, "team_name", None)
+                        # TODO: move to debug or removed entirely
+                        self.log.info("Executor %s belongs to team %s", executor_obj.name, executor_team)
```

Comment:
What is the goal here with `task_team` and `executor_team`? What are we going to do with these 2?

Reply:
Yeah, fair, I was hoping to have it for debugging. But I suppose I can add it in while doing UAT testing and drop it for the merge to main.

Possibly could get _try_to_load_executor(..) to return the team name, because it has to look it up anyway.